source/colors.rst

```diff
@@ -1,62 +1,64 @@
 Style and Colors
 ================
 
-Logos for Dask are available at :doc:`logos`.
-
-For colors we typically choose from somewhere within the palette defined by the
-Dask logo itself, which has an orange gradient.  In particular, we commonly
-choose the following colors: #FDA061, #ECB172, and #D67548.
-
-.. image:: images/dask-colors.svg
-   :alt: Dask colors #FDA061 #ECB172 #D67548
-
-For more involved diagrams we often choose from both primary colors and secondary
-colors:
-
-.. image:: images/dask-branding-guidelines.png
-   :alt: Dask branding guidelines
+These are guidelines for using the Dask logo with community-approved color palettes
```

Comment:
@jsignell wanted to flag that I edited this page quite a bit, all info/screenshots are adapted from these slides of updated brand guidelines
[Dask-Brand Guidelines-R2.pdf](https://github.com/dask/marketing/files/7757258/Dask-Brand.Guidelines-R2.pdf)